docs/requirements.md

```diff
@@ -36,29 +36,33 @@ To support VMs running supported paravirtualized Linux, you require a standard 6
 
 ## RAM
 
-2 GB minimum, 4 GB or more recommended
+2 GB minimum, 4 GB or more recommended.
+
+A fixed amount of RAM is allocated to the control domain (dom0). The optimal amount of RAM for the control domain depends on the workload.
 
 ## Disk space
 
 * Locally attached storage (PATA, SATA, SCSI) with 46 GB of disk space minimum, 70 GB of disk space recommended.
 * SAN via HBA (not through software) when installing with multipath boot from SAN.
 
 
-For a detailed list of compatible storage solutions, see the [Hardware Compatibility List (HCL)](hcl.md).
-Network
+For a detailed list of compatible storage solutions, see the [Hardware Compatibility List (HCL)](hardware.md).
+
+## Network
 
 100 Mbit/s or faster NIC. One or more Gb, or 10 Gb NICs is recommended for faster P2V and export/import data transfers and VM live migration.
 
 We recommend that you use multiple NICs for redundancy. The configuration of NICs differs depending on the storage type. For more information, see the vendor documentation.
 
 XCP-ng requires an IPv4 network for management and storage traffic.
 
-> Notes: Ensure that the time setting in the BIOS of your server is set to the current time in UTC. In some support cases, serial console access is required for debug purposes. When setting up XCP-ng configuration, we recommend that you configure serial console access. For hosts that do not have physical serial port or where suitable physical infrastructure is not available, investigate whether you can configure an embedded management device. For example, Dell DRAC or HP iLO. For more information about setting up serial console access, see CTX228930 - How to Configure Serial Console Access on XenServer and later.
+> Notes: Ensure that the time setting in the BIOS of your server is set to the current time in UTC. In some support cases, serial console access is required for debug purposes. When setting up XCP-ng configuration, we recommend that you configure serial console access. For hosts that do not have physical serial port or where suitable physical infrastructure is not available, investigate whether you can configure an embedded management device. For example, Dell DRAC or HP iLO. For more information about setting up serial console access, see [CTX228930 - How to Configure Serial Console Access on XenServer 7.0 and later](https://support.citrix.com/article/CTX228930).
```

Comment:
I don't know if a tip would be better than a "Note"?

Reply:
Done